weetweet.py

```diff
@@ -763,25 +824,28 @@ def get_twitter_data(cmd_args):
             #The home stream prints you messages as well...
             tweet_data = []
         elif cmd_args[3] == "re":
-            tweet_data = [twitter.statuses.update(status=h.unescape(cmd_args[5]),
-                in_reply_to_status_id=cmd_args[4])]
+            tweet_data = [twitter.statuses.update(status=h.unescape(cmd_args[5]), \
```

Comment:
You do not need a backslash if the arguments are inside [], () or {}
Same for the others below.